modules/EnsEMBL/Web/Object/Gene.pm

```diff
@@ -787,16 +752,13 @@ sub get_homologies {
   $homology_source      = 'ENSEMBL_HOMOLOGUES' unless defined $homology_source;
   $homology_description = 'ortholog' unless defined $homology_description;
   
-  my $geneid   = $self->stable_id;
   my $database = $self->database($compara_db);
-  my %homologues;
-
   return unless $database;
 
-  my $query_member   = $database->get_GeneMemberAdaptor->fetch_by_stable_id($geneid);
-
+  my $query_member   = $database->get_GeneMemberAdaptor->fetch_by_stable_id($self->stable_id);
   return unless defined $query_member;
   
+  my %homologues;
```

Comment:
I am abit confused with this line, you removed the %homologues from line 792 which makes sense but why did you add it back here? It feels like its not being used elsewhere in that function so can be removed unless i missed something.

Reply:
You're right! I think I moved it down on the grounds that it's kinda pointless to initialise a variable just before a conditional return, then forgot to check if the variable was actually used. I'll remove it.